PhotoAlbumSample/app/src/main/res/raw/awsconfiguration.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
You should not be committing this file. Please remove it immediately as it contains sensitive information.